flow/activities/flowable.go

```diff
@@ -247,6 +248,7 @@ func (a *FlowableActivity) CreateNormalizedTable(
 	})
 	defer shutdown()
 
+	a.Alerter.LogFlowInfo(ctx, config.FlowName, "Setting up destination tables")
```

Comment:
why do we have this twice?

Reply:
think it's meant to say conn.StartupSetupNormalizedTables has created tables, beginning ingestion. Wording needs to be updated